test/Microsoft.AspNetCore.Mvc.RazorPages.Test/Infrastructure/PageActionInvokerProviderTest.cs

```diff
@@ -28,15 +27,13 @@ namespace Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure
 {
     public class PageInvokerProviderTest
     {
-        private readonly IHostingEnvironment _hostingEnvironment = Mock.Of<IHostingEnvironment>(e => e.ContentRootPath == "BasePath");
-
         [Fact]
         public void OnProvidersExecuting_WithEmptyModel_PopulatesCacheEntry()
         {
             // Arrange
             var descriptor = new PageActionDescriptor
             {
-                RelativePath = "Path1",
+                RelativePath = "/Path1",
```

Comment:
As mentioned earlier, path w/o leading slash should still work. I don't mean this change should be undone (was previously testing something that should be impossible -- invalid `PageActionDescriptor`s) but please confirm lookups are tested (and succeed) using a path w/o the initial slash.

Reply:
`VirtualRazorProjectFileSystem` that's used in tests supported relative paths without leading slashes, but the default implementation did not. This wasn't a valid test case